conda/exports.py

```diff
@@ -104,8 +104,12 @@
 get_rc_urls = lambda: list(conda.base.context.context.channels)
 get_local_urls = lambda: list(get_conda_build_local_url()) or []
 load_condarc = lambda fn: conda.base.context.reset_context([fn])
-from .exceptions import PaddingError  # NOQA
+from .exceptions import PaddingError, LinkError, CondaOSError, CondaFileNotFoundError  # NOQA
 PaddingError = PaddingError
+LinkError = LinkError
+CondaOSError = CondaOSError
+# clobbering Python's FileNotFoundError - be sure to use module namespacing
```

Comment:
To clarify, will you leave this as CondaFileNotFoundError in 4.3.x, but change to PathNotFoundError in 4.4.x?